src/parser.ts

```
@@ -0,0 +1,1897 @@
+    if (this.check(TokenKind.Then)) {
+      // This is actually an inline if expression used as statement
+      this.advance();
+      const thenExpr = this.parseExpression();
+      this.expect(TokenKind.Else, "Expected 'else' in inline if");
+      const elseExpr = this.parseExpression();
+      return {
+        type: "ExprStmt",
+        expr: {
+          type: "IfExpr",
+          condition,
+          then: thenExpr,
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)